### DISCUSSION TITLE
Different K8sLauncher cpu/gpu resources configuration for different pipelines repo

### DISCUSSION THREAD

**Discussion by abdulvirta** [dagster-io/dagster#9034] [Q&A]
Hi all, how can I specify different cpu/gpu resources in K8sLauncher for different code deployments repo? I have many repo with pipelines with significant differences in terms of requirements. Some needs cpu only, some needs t4 GPU, some need a-100 GPU and so on... Am I understanding this correctly that the K8sLauncher is configured once in the Dagster installation and will be used by all different pipelines repos? 
In which case this seems to be a very weird architecture decision and something that is very limiting. It means if I have one pipeline that requires 4 a-100 GPU machines, that means all other pipelines have to use that much resources. 

Please let me know if what is the recommended way in this case?

Thanks!

**Comment by smackesey** [comment]
cc @johannkm @rexledesma

**Comment by slamer59** [comment]
Maybe you can find a answers from here : 
https://github.com/dagster-io/dagster/discussions/8001

**Reply by abdulvirta** [reply]
Thank you for the reference. Yes I think it should work. I will give it a try.
